xpdan/data_reduction.py

```diff
@@ -0,0 +1,382 @@
+#!/usr/bin/env python
+##############################################################################
+#
+# xpdacq            by Billinge Group
+#                   Simon J. L. Billinge sb2896@columbia.edu
+#                   (c) 2016 trustees of Columbia University in the City of
+#                        New York.
+#                   All rights reserved
+#
+# File coded by:    Timothy Liu
+#
+# See AUTHORS.txt for a list of people who contributed.
+# See LICENSE.txt for license information.
+#
+##############################################################################
+#from dataportal import DataBroker as db
+#from dataportal import get_events, get_table, get_images
+#from metadatastore.commands import find_run_starts
+
+import os
+import warnings
+import datetime
+import yaml
+import numpy as np
+import tifffile as tif
+import matplotlib as plt
+from time import strftime
+from unittest.mock import MagicMock
+
+from .glbl import an_glbl
+from .utils import _clean_info, _timestampstr
+
+from pyFAI.azimuthalIntegrator import AzimuthalIntegrator
+
+# top definition for minimal impacts on the code 
+if an_glbl._is_simulation:
```

Comment:
That shouldn't be too bad, although I haven't worked with ipython profiles before. 
So long as it is running on a linux system then conda should do most of the dependency heavy lifting.
Let me see what I can do, I might have to punt until the next release because of time constraints.